lib/uplink/bucket_attrs_test.go

```diff
@@ -57,6 +58,45 @@ func simpleEncryptionAccess(encKey string) (access EncryptionAccess) {
 	return access
 }
 
+func createPartnerID() (partnerID string, err error) {
+	id, err := uuid.New()
+	if err != nil {
+		return partnerID, err
+	}
+	return id.String(), err
+}
+
+// check that partner bucket attributes are stored and retrieved correctly.
+func TestPartnerBucketAttrs(t *testing.T) {
+	var (
+		access     = simpleEncryptionAccess("voxmachina")
+		bucketName = "mightynein"
+	)
+
+	testPlanetWithLibUplink(t, testConfig{},
+		func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, proj *Project) {
+			_, err := proj.CreateBucket(ctx, bucketName, nil)
+			require.NoError(t, err)
+
+			partnerID, err := createPartnerID()
+			require.NoError(t, err)
+
+			// partner ID set
+			proj.uplinkCfg.Volatile.PartnerID = partnerID
+			got, err := proj.OpenBucket(ctx, bucketName, &access)
+			require.NoError(t, err)
+
+			got, err = proj.OpenBucket(ctx, bucketName, &access)
+			require.NoError(t, err)
+
+			// partner ID NOT set
+			proj.uplinkCfg.Volatile.PartnerID = ""
+			got, err = proj.OpenBucket(ctx, bucketName, &access)
+			require.NoError(t, err)
```

Comment:
Same comment as on line 87.

Reply:
This is return with no partenID set. Previous test case was with a valid partner id set.